src/main/java/net/wasdev/wlp/common/springboot/util/SpringBootThinUtil.java

```diff
@@ -99,8 +98,9 @@ private void thin() throws FileNotFoundException, IOException, NoSuchAlgorithmEx
 					storeEntry(thinJar, entry);
 				}
 			}
-			addLibIndexFileToThinJar(thinJar);
-
+			addLibIndexFileToThinJar(thinJar);			
+		} catch (FileNotFoundException e) {
+			throw new IllegalStateException("Could not access the thin jar file");
```

Comment:
Should the name of the thin jar file be included in this exception? Actually, wondering why we don't just let the FileNotFoundException get returned here? I know Charles asked about a catch, but not really seeing the point of catching a FileNotFoundException and obscuring the actual exception with the IllegalStateException.